src/kernels/raw/session/rawKernel.node.ts

```diff
@@ -22,24 +23,15 @@ input request, translating them, sending them to an IPython kernel over ZMQ, the
 */
 export class RawKernel implements Kernel.IKernelConnection {
     public socket: IKernelSocket & IDisposable;
-    public get statusChanged() {
-        return this.realKernel.statusChanged as any; // NOSONAR
```

Comment:
These were all incorrect, the first argument of the signal event is the sender, 
With this current code, the sender is `realKernel` instead of `this`.
& that causes issues as the Kernel.IKernelConnection the user expects would point to `this`, but infact it ponts to some other object (whith happens to be `this.realKernel`).

Basically its wrong